bot.py

```diff
@@ -1,6 +1,8 @@
 import asyncio
 import discord
 import os
+import csv
```

Comment:
very minor thing but imports should be alphabetical so this would go between asyncio and discord